xml/operations-maintenance-upgrade_maintenance.xml

```diff
@@ -0,0 +1,1834 @@
+
+TASK: [setup] *****************************************************************
+ok: [ardana-cp-mml-m3]
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbstart.yml pb_start_playbook] *****************************************
+ok: [localhost] => {
+    "msg": "Playbook started - ardana-reboot.yml"
+}
+
+msg: Playbook started - ardana-reboot.yml
+
+
+
+...
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbfinish.yml pb_finish_playbook] ***************************************
+ok: [localhost] => {
+    "msg": "Playbook finished - ardana-status.yml"
+}
+
+msg: Playbook finished - ardana-status.yml
+
+PLAY [localhost] **************************************************************
+
+TASK: [pbfinish.yml pb_finish_playbook] ***************************************
+ok: [localhost] => {
+    "msg": "Playbook finished - ardana-reboot.yml"
+}
+
+msg: Playbook finished - ardana-reboot.yml
+
+PLAY RECAP ********************************************************************
+ardana-cp-mml-m3           : ok=389  changed=105  unreachable=0    failed=0
+localhost                  : ok=27   changed=1    unreachable=0    failed=0
+</screen>
+     </step>
+   </procedure>
+   <note>
+     <para>
+       You can reboot more than one control plane node at a time, but only if
+       they are members of different control plane clusters. For example, you
+       could reboot one node out of each of the Openstack controller, database,
+       Swift, monitoring or logging clusters, so long as doing do only reboots
+       one node out of each cluster at the same time.
+     </para>
+   </note>
+
+   <para>
+     When rebooting the first member of the control plane cluster where
+     monitoring services run, the <literal>monasca-thresh</literal> service can sometimes fail
+     to start up in a timely fashion when the node is coming back up after
+     being rebooted. This can cause <filename>ardana-reboot.yml</filename> to fail.
+     See below for suggestions on how to handle this problem.
+   </para>
+
+   <procedure>
+     <title>Getting <literal>monasca-thresh</literal> Running After an <filename>ardana-reboot.yml</filename> Failure</title>
+       <para>
+         If the <filename>ardana-reboot.yml</filename> playbook failed because
+         <literal>monasca-thresh</literal> didn't start up in a timely fashion
+         after a reboot, you can retry starting the services on the node using
+         the <filename>_ardana-post-reboot.yml</filename> playbook for the node.
+         This is similar to the manual handling of the deployer reboot, since
+         the node has already successfully rebooted onto the new kernel, and
+         you just need to get the required services running again on the node.
+       </para>
+       <para>
+         It can sometimes take up to 15 minutes for the <literal>monasca-thresh</literal>
+         service to successfully start in such cases.
```

Comment:
```suggestion
     <title>Getting <systemitem class="service">monasca-thresh</systemitem> Running After an <filename>ardana-reboot.yml</filename> Failure</title>
       <para>
         If the <filename>ardana-reboot.yml</filename> playbook failed because
         <systemitem class="service">monasca-thresh</systemitem> didn't start up in a timely fashion
         after a reboot, you can retry starting the services on the node using
         the <filename>_ardana-post-reboot.yml</filename> playbook for the node.
         This is similar to the manual handling of the deployer reboot, since
         the node has already successfully rebooted onto the new kernel, and
         you just need to get the required services running again on the node.
       </para>
       <para>
         It can sometimes take up to 15 minutes for the <systemitem class="service">monasca-thresh</systemitem>
         service to successfully start in such cases.
```